attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/builder/AttributeFilterBuilderImpl.java

```diff
@@ -0,0 +1,25 @@
+package org.hypertrace.core.attribute.service.builder;
+
+import static org.hypertrace.core.documentstore.expression.operators.RelationalOperator.EQ;
+
+import org.hypertrace.core.documentstore.expression.impl.ConstantExpression;
+import org.hypertrace.core.documentstore.expression.impl.IdentifierExpression;
+import org.hypertrace.core.documentstore.expression.impl.RelationalExpression;
+import org.hypertrace.core.documentstore.expression.type.FilterTypeExpression;
+
+public class AttributeFilterBuilderImpl implements AttributeFilterBuilder {
+  private static final String ID_SUB_DOC_PATH = "id";
```

Comment:
guessing these are already defined elsewhere, would be good to avoid duplicating.